src/index.js

```diff
@@ -1,10 +1,31 @@
+/* eslint-disable import/first */
+
+// Conflicting Components
+import Button from 'react-bootstrap/Button';
+import ButtonDeprecated from './Button';
+
+import Dropdown from 'react-bootstrap/Dropdown';
+import DropdownDeprecated from './Dropdown';
+
+import Tabs from 'react-bootstrap/Tabs';
+import TabsDeprecated from './Tabs';
+
+// Add deprecated components as subcomponents of their replacements
+Button.Deprecated = ButtonDeprecated;
```

Comment:
This seems like it may cause confusion... if I want the deprecated Button, I need to import Button from this index file (rather than the sub-directory), and only then will I find the `Deprecated` implementation hanging off of it... Would it be better for the `Button` sub-directory to publish the react-bootstrap Button and in `Button/index.js` attach this Deprecated version there?  That way when it's imported here and re-exported, it just works and is consistent.

Reply:
Put another way - the button sub-directory can be responsible for its exports - all of them.  If it wants a `Deprecated` button hanging off of it, it can do it itself and this file (or any other consumers) will be none the wiser about where _either_ implementation came from.

Reply:
Good point, I'll refactor to your suggestion.

Reply:
Done! Please take a look.